src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,54 @@
+.nav {
+  margin: 0;
+  padding: 0;
+
+  &__bar {
+    display: flex;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+  }
+
+  &__links {
+    display: flex;
+    height: 60px;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    font-size: 12px;
+    text-transform: uppercase;
```

Comment:
in order not to write styles like `margin: 0; padding: 0;` for your list (ul) with "nav__bar" or ` text-decoration: none;` for `a` tag you can create a reset.scss file and don't write the same reset styles each time
for example

```
body,
h1,
h2,
h3,
h4,
p {
  margin: 0;
}

ul {
  margin: 0;
  padding: 0;
  list-style-type: none;
}

a {
  text-decoration: none;
}
```

after you create this file don't forget do delete unnecessary styles from your scss blocks